.github/workflows/perf.yml

```
@@ -3,9 +3,9 @@ name: Performance Test
 on:
   pull_request:
   issue_comment:
-  schedule:
+  #schedule:
```

Please don't modify the `.github` folder:

```suggestion
  schedule:
```